src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/EventValidationService.java

```diff
@@ -352,9 +353,9 @@ public void validateJudgementsHasJurisdiction(CaseData caseData, boolean partOfM
         }
     }
 
-    public List<String> validateDepositRefunded(CaseData caseData) {
+    public List<String> validateDepositOrder(CaseData caseData) {
```

Comment:
sonarlint's flagged this method as being too cognitively complex, could try and refactor it to reduce its size and complexity